DataFormats/L1TParticleFlow/interface/jets.h

```diff
@@ -13,6 +13,8 @@ namespace l1ct {
     pt_t hwPt;
     glbeta_t hwEta;
     glbphi_t hwPhi;
+    z0_t z0;
```

Comment:
```suggestion
    z0_t hwZ0;
```
(for consistency with the naming of the rest of the variables in l1ct objects)